api/time_bucket.md

```diff
@@ -1,58 +1,59 @@
-## time_bucket() 
+# time_bucket()
+The `time_bucket` function is based on the standard PostgreSQL `date_trunc`
+function. It allows for arbitrary time intervals instead of the second, minute,
+and hour intervals provided by `date_trunc`. The return value is the bucket's
+start time.
```

Comment:
```suggestion
The `time_bucket` function is similar to the standard PostgreSQL `date_trunc`
function. Unlike `date_trunc`, it allows for arbitrary time intervals instead of 
second, minute, and hour intervals. The return value is the bucket's
start time.
```